Symfony/CS/Tests/Fixer/Contrib/ShortEchoTagFixerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class ShortEchoTagFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideClosingTagExamples
+     */
+    public function testOneLineFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideClosingTagExamples()
+    {
+        return array(
+            array('<?php echo \'Foo\';', '<?= \'Foo\';'),
+            array('<?php echo \'Foo\'; ?> PLAIN TEXT', '<?= \'Foo\'; ?> PLAIN TEXT'),
+            array('PLAIN TEXT<?php echo \'Foo\'; ?>', 'PLAIN TEXT<?= \'Foo\'; ?>'),
```

Comment:
Another test needing to be added:

```php
<?=foo();
```

 should be turned into

```php
<?php echo foo();
```

Your current code turns it into this different code (calling a non-existent function instead of echoing the return value of the function):


```php
<?php echofoo();
```